<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tlinsWEnvironment</class>
 <widget class="QWidget" name="tlinsWEnvironment">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Temperature</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Real Humidity</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="tlinsWDecimalValue" name="realHumidity" native="true"/>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>[C]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Ambient pressure</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>[hPa]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="tlinsWDecimalValue" name="temperature" native="true"/>
   </item>
   <item row="0" column="1">
    <widget class="tlinsWDecimalValue" name="ambientPressure" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Effective color</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="tlinsWDecimalValue" name="effectiveColor" native="true"/>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tlinsWDecimalValue</class>
   <extends>QWidget</extends>
   <header location="global">tlinswdecimalvalue.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
